Build and Publish Docker Image (latest) to dockerhub (Closes #727) (#728)
mosdef-hub/mbuild: README.md

}
```

#### Quick Start with Docker
To use `mbuild` in a jupyter-notebook that runs from a docker container with all the dependencies installed use the following command:

```bash
$ docker pull mosdef/mbuild:latest
$ docker run -it --name mbuild -p 8888:8888  mosdef/mbuild:latest jupyter notebook --ip="*" --allow-root
```

> Alternatively, you can also start a bash shell directly:

Reviewer: ```suggestion
Alternatively, you can also start a Bourne shell directly:
```

> ```bash

Reviewer: ```suggestion
```sh
```

$ docker run -it --name mbuild mosdef/mbuild:latest
```

> Note: Instead of using `latest`, you can use the image `mosdef/mbuild:stable` for stable released version of `mbuild`.

Reviewer: ```suggestion
Note: Instead of using `latest`, you can use the image `mosdef/mbuild:stable` for most recent stable release of `mbuild`.
```

}
```

#### Quick Start with Docker
To use `mbuild` in a jupyter-notebook that runs from a docker container with all the dependencies installed use the following command:

```sh
$ docker pull mosdef/mbuild:latest
$ docker run -it --name mbuild -p 8888:8888 mosdef/mbuild:latest\

>   /opt/conda/envs/mbuild-docker/bin/jupyter notebook --no-browser --ip="0.0.0.0"

Reviewer: Yeah, no browser, you run the server in the container, and expose ports on both sides, then paste that url into your browser on your machine. You might need to add `—ip=“0.0.0.0”`

>   /opt/conda/envs/mbuild-docker/bin/jupyter notebook --no-browser --ip="0.0.0.0"

Reviewer 2: Ok it worked. I think I couldn't connect since had a different notebook server already running on port 8888. 
This is pretty cool!

>   /opt/conda/envs/mbuild-docker/bin/jupyter notebook --no-browser --ip="0.0.0.0"

Author: Can you show the logs from the notebook server?

```

Alternatively, you can also start a Bourne shell directly:
```sh

> $ docker run -it --name mbuild mosdef/mbuild:latest

Reviewer: Yeah it’s just in the container. You can mount an external dir with the -v I think. I think it just needs to be explained a bit more.

> $ docker run -it --name mbuild mosdef/mbuild:latest

Author: Its up to to user to decide whether or not he/she wants to mount a user volume. Since this is a quickstart guide, let's leave it like it is. We should add a docker tutorial separately in the documentation.

> $ docker run -it --name mbuild mosdef/mbuild:latest

Reviewer 3: Aside from anything super specific to this tutorial I would just point users to external Docker tutorials, the internet is full of them and some of them are pretty decent ([here](https://docker-curriculum.com/) and [here](https://www.youtube.com/watch?v=zJ6WbK9zFpI&feature=youtu.be), for example).
